backend/app/api/endpoints/knowledge.py

```
@@ -1556,3 +1557,71 @@ def get_document_chunk(
+# Create a dedicated router for v1/knowledge-base endpoints
+knowledge_router = APIRouter()
+
+
+@knowledge_router.get("/list")
+@trace_sync("list_knowledge_bases", "knowledge.api")
+def list_knowledge_bases(
+    scope: str = Query(
+        default="all",
+        description="Scope of knowledge bases to return: 'personal' or 'all'",
+    ),
+    auth_context: AuthContext = Depends(get_auth_context),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)